### ISSUE TITLE
Parity account action icons lack tooltip and/or labels

### ISSUE THREAD

**Issue by miohtama** [paritytech/parity#5443]
It is unclear what these icons mean - only way to find out is to dare to click them 

<img width="880" alt="image" src="https://cloud.githubusercontent.com/assets/49922/24931414/d33ee234-1f15-11e7-9ed0-4241a6992c78.png">

**Comment by 5chdn** [comment]
which version?

**Comment by tomusdrw** [comment]
That's what it looks like on smaller screens I suppose. Are you zoomed in? Is your window maximized?
That said I think it would be good to add at least some tooltips on small screens.

**Comment by miohtama** [comment]
This is OSX / Firefox / 1024 px window.

**Comment by 5chdn** [comment]
This issue is labelled with _bug_: The client fails to follow expected behavior. It's assigned to @ngotchac but not linked to a milestone.

Since there's a patch available, I added it to the 1.7 milestone. Please, don't forget to close it once #5562 is merged. Please, also check if #5432 is a duplicate of this issue.
